runtime/Cargo.toml

```diff
@@ -70,6 +70,11 @@ jsonrpc = { version = "0.13.0", features = ["simple_uds"] }
 tempfile = "3.4.0"
 tendermint-testgen = "0.30.0"
 
+[features]
+default = []
+# Enables debug-level logging in release builds.
+debug-logging = ["slog/max_level_debug", "slog/release_max_level_debug"]
```

Comment:
Yes, I just wanted to make sure this can't get overriden by some change in default. But maybe they will never change that so just having the first feature will always be enough.